### ISSUE TITLE
Instagram

### ISSUE THREAD

**Issue by simonbcn** [extesy/hoverzoom#960]
## Description of the problem
It doesn't work on Instagram. 

## Url of the page where the problem occurs (not the image url)
https://www.instagram.com/

## OS version, extension version
Arch Linux
Google Chrome 102.0.5005.115
Hover Zoom+ 1.0.193

This is the console output with `debug` enabled:
```
DevTools no ha podido cargar el mapa de la fuente: No se ha podido cargar contenido de chrome-extension://pccckmaobkjjboncdfnnofkonhgpceea/js/hls.js.map: Error del sistema: net::ERR_BLOCKED_BY_CLIENT
[Violation] Added non-passive event listener to a scroll-blocking <some> evento. Consider marking event handler as 'passive' to make the page more responsive. See <URL>
[Violation] Added non-passive event listener to a scroll-blocking <some> evento. Consider marking event handler as 'passive' to make the page more responsive. See <URL>
[Violation] Added non-passive event listener to a scroll-blocking <some> evento. Consider marking event handler as 'passive' to make the page more responsive. See <URL>
.......
[Violation] Added non-passive event listener to a scroll-blocking <some> evento. Consider marking event handler as 'passive' to make the page more responsive. See <URL>
[Violation] Added non-passive event listener to a scroll-blocking <some> evento. Consider marking event handler as 'passive' to make the page more responsive. See <URL>
[Violation] Added non-passive event listener to a scroll-blocking <some> evento. Consider marking event handler as 'passive' to make the page more responsive. See <URL>
jquery-3.2.1.js:5062 [Violation] Added synchronous DOM mutation listener to a 'DOMNodeInserted' event. Consider using MutationObserver to make the page more responsive.
add @ jquery-3.2.1.js:5062
(anónimo) @ jquery-3.2.1.js:4962
each @ jquery-3.2.1.js:362
each @ jquery-3.2.1.js:157
on @ jquery-3.2.1.js:4961
on @ jquery-3.2.1.js:5570
bind @ jquery-3.2.1.js:10168
bindEvents @ hoverzoom.js:1949
init @ hoverzoom.js:2902
applyOptions @ hoverzoom.js:1868
(anónimo) @ hoverzoom.js:1906
hoverzoom.js:11 shortcode:Ce1XjhFjGjn/liked_by
hoverzoom.js:11 extractSharedData
hoverzoom.js:11 extractHookedData
hoverzoom.js:11 shortcode:Ce5zbt7MjRz/liked_by
hoverzoom.js:11 extractSharedData
hoverzoom.js:11 extractHookedData
hoverzoom.js:29 prepareImgLinks: 7.529052734375 ms
hoverzoom.js:11 shortcode:Ce1XjhFjGjn/liked_by
hoverzoom.js:11 extractSharedData
hoverzoom.js:11 extractHookedData
hoverzoom.js:11 shortcode:Ce5zbt7MjRz/liked_by
hoverzoom.js:11 extractSharedData
hoverzoom.js:11 extractHookedData
hoverzoom.js:29 prepareImgLinks: 6.95703125 ms
D_4bEktlZQ-.js?_nc_x=Ij3Wp8lg5Kz:212 [Violation] 'message' handler took 164ms
[Violation] Forced reflow while executing JavaScript took 47ms
hoverzoom.js:11 shortcode:Ce4BORbtsQh/liked_by
hoverzoom.js:11 extractSharedData
hoverzoom.js:11 extractHookedData
hoverzoom.js:11 shortcode:CezQieXAVss/liked_by
hoverzoom.js:11 extractSharedData
hoverzoom.js:11 extractHookedData
hoverzoom.js:29 prepareImgLinks: 8.150146484375 ms
hoverzoom.js:11 cancelImageLoading
hoverzoom.js:11 closeHoverZoomImg(undefined)
hoverzoom.js:11 closeHoverZoomImg(undefined)
hoverzoom.js:11 cancelImageLoading
hoverzoom.js:11 closeHoverZoomImg(undefined)
```

**Comment by TraceGroot** [comment]
Is there any update on when this will be fixed?

**Comment by extesy** [comment]
Not yet. Instagram makes it very hard to access original images.

**Comment by GrosPoulet** [comment]
Hello @TraceGroot 
HZ+ zooms IG images & videos since latest release (1.0.202).

**Comment by extesy** [comment]
Closing as fixed in `1.0.202`.
